website/static/js/registrationUtils.js

```diff
@@ -779,7 +781,67 @@ RegistrationEditor.prototype.save = function() {
     }
     return true;
 };
+/**
+ * Makes ajax request for a project's contributors
+ */
+RegistrationEditor.prototype.makeContributorsRequest = function() {
+    var self = this;
+    var contributorsUrl = window.contextVars.node.urls.api + 'contributors_abbrev/';
+    return $osf.ajaxJSON('get', contributorsUrl);
+};
+/**
+ * Returns the `user_fullname` of each contributor attached to a node.
+ **/
+RegistrationEditor.prototype.getContributors = function() {
+    var self = this;
+    var contributorNames = [];
+    self.makeContributorsRequest()
+        .done(function(data) {
+            $.map(data.contributors, function(val, i) {
+                contributorNames.push(val.user_fullname);
+            });
+        }).fail(function() {
+            $osf.growl('Could not retrieve contributors.', 'Please refresh the page or ' +
+                       'contact <a href="mailto: support@cos.io">support@cos.io</a> if the ' +
+                       'problem persists.');
+        });
+    return contributorNames;
+};
+/**
+ * Opens a bootbox dialog with a checkbox list of each contributor
+ * the user has the option to import all contributors or to select
+ * each one individually.
+ **/
+RegistrationEditor.prototype.authorDialog = function() {
```

Comment:
When we're not behind schedule and rushed we should generalize this pattern of KO rendering into bootboxes as a reusable helper.